tensorflow/compiler/xla/service/tuple_simplifier.cc

```diff
@@ -30,10 +30,19 @@ limitations under the License.
 
 namespace xla {
 
+TupleSimplifier::TupleSimplifier(bool exclude_entry_computation) :
+    exclude_entry_computation_(exclude_entry_computation) {}
+
 StatusOr<bool> TupleSimplifier::Run(HloModule* module) {
   // Initially add all GTE and Tuple instructions to the worklist.
   std::queue<HloInstruction*> worklist;
   for (auto* computation : module->computations()) {
+    LOG(INFO) << "*** Computation is " << computation->name() << " at " << (void*)computation;
```

Comment:
Please use VLOG(N) instead of LOG(INFO).  Although I'm not sure these are actually useful to keep at all?  If you want to keep them, please try to follow the style in other passes wrt e.g. the `***` headers and so on.